src/main/scala/outwatch/package.scala

```diff
@@ -0,0 +1,35 @@
+import cats.effect.IO
+import outwatch.Sink.SubjectSink
+import outwatch.advanced._
+
+package object outwatch {
+
+  type >-->[-I, +O] = Pipe[I, O]
+  val >--> = Pipe
+
+  type Handler[A] = Pipe[A, A]
```

Comment:
If we do this, we can call `mapSink` on a Handler and get back a `Pipe[A, B]` not sure if that makes sense.

Reply:
Yes, I see what you mean. How do you feel about changing `Handler` to have two type parameters? I shouldn't break almost any client code, Handler type is not typically used much in client code.

Reply:
You're right, I think the solution below is probably the best :)